HotSpot/View Controllers/BookedParkingSpotsViewController.m

```diff
@@ -29,10 +29,13 @@ @implementation BookedParkingSpotsViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    // initializing and reloading tableView
     self.bookedTableView.dataSource = self;
     self.bookedTableView.delegate = self;
     self.bookedTableView.rowHeight = 134;
-    [self.bookedTableView reloadData]; 
+    [self.bookedTableView reloadData];
```

Comment:
i don't think you need to call reloadData from viewDidLoad. did you try removing it to see if the row height is updated without calling it?

Reply:
Yea, I didn't need it. Thanks!